src/TimeSeriesInsights/examples/Get-AzTimeSeriesInsightsEnvironment.md

```diff
@@ -78,3 +78,31 @@ Type                         : Microsoft.TimeSeriesInsights/Environments
 
 This command lists all time series insights environments in a resource group.
 
+### Example 3: Get a time series insights environments by object
```

Comment:
Should be environment instead of environments

Reply:
Thanks, Done